app/models/contact.rb

```diff
@@ -1,15 +1,19 @@
 class Contact < ApplicationRecord
   belongs_to :contactable, polymorphic: true, optional: true
 
-  validates :name, presence: true, if: :department?
+  validates :last_name, presence: true, if: :department?
```

Comment:
departments don't have last names.
Just keep the name field. And add first_name and last_name for the human beeings